BookReader/BookReader.js

```diff
@@ -341,68 +493,26 @@ BookReader.prototype.init = function() {
         return false;
     });
 
-    //-------------------------------------------------------------------------
-    // Setup sound manager for read-aloud
-    if (this.isSoundManagerSupported) this.setupSoundManager();
-
+    this.addEventListener('stop', function(e, br) {
+        br.autoStop();
+    });
 
-    $(document).trigger("BookReader:PostInit");
+    this.trigger('PostInit');
 
     this.init.initComplete = true;
 }
 
-
-//______________________________________________________________________________
-
-/**
- * Throttle function
- * @see https://remysharp.com/2010/07/21/throttling-function-calls
- */
-BookReader.prototype.throttle = function(fn, threshhold, delay) {
-    threshhold || (threshhold = 250);
-    var last,
-        deferTimer;
-    if (delay) last = +new Date;
-    return function () {
-      var context = this;
-      var now = +new Date,
-          args = arguments;
-      if (last && now < last + threshhold) {
-        // hold on to it
-        clearTimeout(deferTimer);
-        deferTimer = setTimeout(function () {
-          last = now;
-          fn.apply(context, args);
-        }, threshhold);
-      } else {
-        last = now;
-        fn.apply(context, args);
-      }
-    };
+BookReader.prototype.trigger = function(name, props) {
```

Comment:
Suggestion: If you’re going to follow the browser EventTarget interface, do it for all three methods

Reply:
Okay. I changed them to .dispatchEvent

Reply:
Update. I changed them all to `.bind, .unbind, .trigger`, because the rest of the code uses that a lot, and the other names felt out of place.